lib/fog/ovirt/requests/compute/v4/add_volume.rb

```diff
@@ -13,11 +13,11 @@ def add_volume(id, options = {})
             disk_attachments_service.add(disk)
           end
 
-          # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity
+          # rubocop:disable Metrics/AbcSize
           def add_options_defaults(options)
             options = options.dup
             search = options[:search] || format("datacenter=%<datacenter>s", :datacenter => datacenter)
-            options[:bootable] ||= "true"
+            options[:bootable] = options.delete(:bootable) == "true"
```

Comment:
The behavior before was to set to "true" when `bootable` was not set. here it would be `nil == "true"` resulting in false. Is that intentional?

Reply:
yes, in case bootable is checked , options[:bootable] should be true and not "true" 
and in case bootable is unchecked options[:bootable] should be false

Reply:
yes, in case bootable is checked , options[:bootable] should be true and not "true"
and in case bootable is unchecked options[:bootable] should be false

Reply:
and nil should result in false too.